lti_consumer/lti_1p3/extensions/rest_framework/serializers.py

```diff
@@ -374,3 +376,89 @@ class LtiDlImageSerializer(serializers.Serializer):
     thumbnail = LtiDLIconPropertySerializer(required=False)
     width = serializers.IntegerField(min_value=1, required=False)
     height = serializers.IntegerField(min_value=1, required=False)
+
+
+class LtiContextSerializer(serializers.Serializer):
+    """
+    Serializer for a LTI Context
+    """
+    id = UsageKeyField()
+
+
+class LtiNrpsContextMemberBasicSerializer(serializers.Serializer):
+    """
+    Non PII fields serializer for Context Member.
+    """
+    status = serializers.SerializerMethodField()
+    user_id = serializers.SerializerMethodField()
+    roles = serializers.SerializerMethodField()
+
+    def get_user_id(self, user):
+        """
+        Get external user id for the user.
+        """
+        externalid, _ = compat.get_or_create_externalid(user)
+        return serializers.UUIDField().to_representation(externalid.external_user_id)
+
+    def get_status(self, user):
+        """
+        Get user status string value.
+        """
+        return 'Active' if user.is_active else 'Inactive'
```

Comment:
@shimulch Exclude the inactive users from the API. The spec doesn't require us to return them, and that will be somewhat useless data.

Reply:
Now it will return only `Active` as per the data api.